CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 See also https://github.com/neo4j/neo4j-python-driver/wiki for more details.
 
+## Version 5.6
```

Comment:
isn't this too early?

Reply:
:shrug: I could also put "next release" into the headline and change it once I release it. Might be better people looking into it.